test/support/pages/HeaderPage.js

```diff
@@ -128,8 +135,19 @@ define(function (require) {
           return;
         });
       });
-    }
+    },
 
+    getSpinnerDone: function getSpinnerDone() {
+      var self = this;
+      return this.remote
+      .setFindTimeout(defaultTimeout * 10)
+      .findByCssSelector('span.spinner.ng-hide');
+      // .then(function () {
+      //   return self.remote
+      //   .setFindTimeout(defaultTimeout * 10)
+      //   .findByCssSelector('div.spinner.large.ng-hide');
+      // });
```

Comment:
I'd like to leave this for now.  I don't think we've come up with the best solution yet for knowing when Kibana is done loading various bits of data on each tab.  I'm not a fan of the tryForTime approach that checks multiple times to see if/when we get the expected result.